doc/source/reference/c-api/ufunc.rst

```diff
@@ -10,7 +10,7 @@ UFunc API
 Constants
 ---------
 
-.. c:var:: UFUNC_ERR_{HANDLER}
+.. :ref: UFUNC_ERR_{HANDLER}
```

Comment:
Does this work? I'm suspicious that this becomes a comment, and doesn't appear at all.

Reply:
Yeah, this does not work: https://16364-908607-gh.circle-artifacts.com/0/doc/build/html/reference/c-api/ufunc.html#constants.

It's possible you want

```suggestion
.. ref:: UFUNC_ERR_{HANDLER}
```

but I won't be too hopeful that that exists either.

Reply:
You're right, `ref::` directive does not exists. And these are not referenced, so should they be marked as section title?
```suggestion
UFUNC_ERR_{HANDLER}
^^^^^^^^^^^^^^^^^^^
```

Reply:
Can you make this an indented section?
```suggestion
``UFUNC_ERR_{HANDLER}``
```

Reply:
It works. This and another `:ref:`-related comment (https://github.com/numpy/numpy/pull/16370#discussion_r504555662) don't depend on sphinx 3, so I'll create another PR.